src/test/kotlin/uk/gov/justice/digital/hmpps/hmppschallengesupportinterventionplanapi/resource/UpsertDecisionActionIntTest.kt

```diff
@@ -376,7 +377,7 @@ class DecisionActionIntTest : IntegrationTestBase() {
       csipRecord,
       AuditEventAction.UPDATED,
       setOf(AffectedComponent.DecisionAndActions),
-      "Updated decision and actions record conclusion changed from 'a comprehensive conclusion' to 'a conclusion', " +
+      "Updated decision and actions conclusion changed from 'a comprehensive conclusion' to 'a conclusion', " +
```

Comment:
This reads a bit strangely, I assume a part of `Updated decision and actions conclusion` is a property name? Might be worth wrapping this in single quotes. Is `Updated` your action or part of a property name?